src/main.cpp

```diff
@@ -2019,52 +2019,60 @@ bool AcceptToMemoryPool(
 bool GetTimestampIndex(unsigned int high, unsigned int low, bool fActiveOnly,
     std::vector<std::pair<uint256, unsigned int> > &hashes)
 {
-    if (!fTimestampIndex)
-        return error("Timestamp index not enabled");
-
-    if (!pblocktree->ReadTimestampIndex(high, low, fActiveOnly, hashes))
-        return error("Unable to get hashes for timestamps");
-
+    if (!fTimestampIndex) {
+        LogPrint("rpc", "Timestamp index not enabled");
```

Comment:
Should all these `LogPrint`'s use a category of "net"?

Reply:
The `GetSpentIndex()` callers are within the RPC code: https://github.com/zcash/zcash/search?q=GetSpentIndex
and similar for the other 3 `Get*` functions that we're modifying. So, for example, if the `getrawtransaction` RPC doesn't return the "spent" information as expected, and you would like to know why, you can enable the `rpc` logging category, re-run the `getrawtransaction` RPC and look in the log file to see why.

The `net` logging category relates to the P2P network.

Another option I considered would be for these to have their own new category (`insight` perhaps, although that doesn't refer to lightwalletd), but then I thought these message fit pretty naturally in the `rpc` category.

Reply:
The PR description above mentions `-debug=net` which is why I asked.

Reply:
Oh I'm sorry, you're right, fixed!